### PR TITLE
feat: update helm spec with autocomplete-tools for cobra

### PR DESCRIPTION
**Pull request by dirien**
Signed-off-by: Engin Diri <engin.diri@mail.schwarz>

**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Update of the helm cli using the autocomplete-tools for cobra

**What is the current behavior? (You can also link to an open issue here)**
-

**What is the new behavior (if this is a feature change)?**

Update of the helm cli using the autocomplete-tools for cobra

**Additional info:**

### CODE REVIEW

**Review by mschrage — CHANGES_REQUESTED** [review]
This looks awesome! Thank you for updating the spec.

Can you make make sure that the arguments which take a filepath, eg. for `--keyring`, include `template: "filepaths"` ?

**Comment by dirien** [comment]
Hi @mschrage, thanks for the feedback! 

I use the the autocomplete tool for cobra and want to use the autopush action in the future! 

Is this a manual rework I have to do every time?

**Comment by fedeci** [comment]
Hi @dirien, you won't need to do this manual stuff every time because the autocomplete action is using a merge tool retaining manually set options.

File: src/helm.ts [hunk 1]
```diff
@@ -1318,99 +2106,170 @@ const completionSpec: Fig.Spec = {
...
     },
     {
       name: "--registry-config",
       description: "Path to the registry config file",
       isPersistent: true,
       args: {
-        name: "REGISTRY_CONFIG",
-        template: "filepaths",
+        name: "registry-config",
+        default: "/Users/dirien/Library/Preferences/helm/registry/config.json",
```
Comment (SeparateRecords) [inline src/helm.ts:2217]: Can you replace all occurrences of `/Users/dirien` with `~`?

**Review by SeparateRecords — CHANGES_REQUESTED** [review]
Looking good! Just one extra change on top of the `template: "filepaths"` stuff.

If you want I'm happy to do it for you 🙂

**Comment by dirien** [comment]
Ok did my best to find all `template: "filepaths"` arguments. Feel free to add, if you found one. This is a huge spec! 😅

**Comment by SeparateRecords** [comment]
LGTM! Thank you @dirien 😊

**Pull request merged**